tests/Microsoft.Identity.Test.Unit/WebUITests/HttpListenerInterceptorTests.cs

```diff
@@ -90,31 +90,55 @@ await AssertException.TaskThrowsAsync<OperationCanceledException>(
         [TestMethod]
         public async Task ValidateHttpListenerRedirectUriAsync()
         {
-            HttpListenerInterceptor listenerInterceptor = new HttpListenerInterceptor(
-                Substitute.For<ILoggerAdapter>());
+            HttpListenerInterceptor listenerInterceptor = new(Substitute.For<ILoggerAdapter>());
 
             int port = FindFreeLocalhostPort();
+            listenerInterceptor.TestBeforeStart = (url) => Assert.AreEqual($"http://localhost:{port}/TestPath/", url);
 
-            listenerInterceptor.TestBeforeStart = (url) => Assert.AreEqual(@"http://localhost:" + port + @"/TestPath/", url);
-
-            // Start the listener in the background
+            // Start listener in the background
             Task<Uri> listenTask = listenerInterceptor.ListenToSingleRequestAndRespondAsync(
                 port,
                 "/TestPath/",
```

Comment:
The string "/TestPath/" is used more than once. Should it be defined as a constant?